roles/index-lifecycle-management/tasks/main.yml

```diff
@@ -11,5 +11,19 @@
       Content-Type: application/json
       kbn-xsrf: true
   with_items:
-    "{{ environments }}"
+    "{{ application_environments }}"
+  no_log: True
+
+- name: Set index-lifecycle-management gateway policies for environments
```

Comment:
We should probably have a single `put` and iterate through the different policies going forwards but this is fine for now